graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,353 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import ot
+import numpy as np
+from sklearn.utils import check_array
+
+from .base import BaseAlign
+from .sign_flips import SignFlips
+from .orthogonal_procrustes import OrthogonalProcrustes
+
+
+class SeedlessProcrustes(BaseAlign):
+    """
+    Implements an algorithm that matches two datasets using an orthogonal
+    matrix. Unlike OrthogonalProcrustes, this does not use a matching between
```

Comment:
properly ref to `OrthogonalProcrustes`, there should be examples in the embed stuff of how to do so

Reply:
Done.